data/mods/Russ_Extra_Stuff/items/armor.json

```diff
@@ -0,0 +1,138 @@
+[{
+        "type" : "ARMOR",
+        "id" : "adv_power_armor_frame",
+        "name" : "adv. power armor hauling frame",
+        "description" : "A gyroscopic hauling backpack built for use with powered armor.  It has many extra straps and a telescoping superalloy frame, allowing for increased storage.",
+        "weight" : 1640,
+        "color" : "light_blue",
+        "price" : 100000,
+        "material" : ["superalloy", "null"],
+        "volume" : 100,
+        "cutting" : 0,
+        "warmth" : 0,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "to_hit" : 1,
+        "power_armor" : true,
+        "storage" : 240,
+        "encumbrance" : 4,
+        "bashing" : 1,
+        "symbol" : "[",
+        "flags" : ["WATERPROOF","STURDY","BELTED"],
+        "coverage" : 100,
+        "material_thickness" : 10
+    },
+{
+        "type" : "TOOL_ARMOR",
+        "id" : "shield_generator",
+        "name" : "shield generator (off)",
+        "weight" : 840,
+        "color" : "light_blue",
+        "covers" :["TORSO","HEAD","LEGS","ARMS","HANDS","EYES","MOUTH","FEET"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "o",
+        "description" : "Your typical shield generator, reminiscent of the 90's game Out Of This World.  It runs on batteries.  It has a radiation compensation unit built in, but running it generates a lot of heat. (activate again)",
+        "price" : 100000,
+        "material" : ["steel", "null"],
+        "volume" : 2,
+        "cutting" : 0,
+        "warmth" : 0,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 0,
+        "bashing" : 0,
+	   "piercing" : 0,
+        "flags" : ["WAIST","WATERPROOF","STURDY","OUTER","OVERSIZE"],
+        "coverage" : 0,
+        "material_thickness" : 2,
+"max_charges": 100,
+    "initial_charges": 100,
+    "charges_per_use": 1,
+    "turns_per_charge": 0,
+    "ammo": "battery",
```

Comment:
I dont use UPS, is there an entry i can use as an example?  Thx
On Aug 4, 2015 9:54 AM, "Coolthulhu" <notifications@github.com> wrote:

> In data/mods/Russ_Extra_Stuff/items/armor.json
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13075#discussion_r36189746>
> :
>
> > +        "volume" : 2,
> > +        "cutting" : 0,
> > +        "warmth" : 0,
> > +        "phase" : "solid",
> > +        "environmental_protection" : 0,
> > +        "encumbrance" : 0,
> > +        "bashing" : 0,
> > +	   "piercing" : 0,
> > +        "flags" : ["WAIST","WATERPROOF","STURDY","OUTER","OVERSIZE"],
> > +        "coverage" : 0,
> > +        "material_thickness" : 2,
> > +"max_charges": 100,
> > +    "initial_charges": 100,
> > +    "charges_per_use": 1,
> > +    "turns_per_charge": 0,
> > +    "ammo": "battery",
>
> This should probably run off UPS. Most advanced electrical equipment does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13075/files#r36189746>.
>